app/design/adminhtml/default/default/template/boltpay/form.phtml

```diff
@@ -24,15 +24,77 @@
  * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
  */
 ?>
+<?php /* @var $this Bolt_Boltpay_Block_Checkout_Boltpay */?>
 <?php
+$quote = Mage::getSingleton('checkout/session')->getQuote();
+$publishableKey = $this->getPublishableKey(false);
+$theme = Mage::getStoreConfig('payment/boltpay/theme');
 $_code = $this->getMethodCode();
-$fieldName = Mage::helper('boltpay')->__('Bolt Transaction Reference');
+$fieldName = Mage::helper('boltpay')->__('Process Order Through Bolt');
 ?>
 <ul id="payment_form_<?php echo $_code ?>" style="display:none">
     <li>
         <div class="input-box">
-            <label for="<?php echo $_code ?>_transaction_reference"><?php echo $fieldName ?> <span class="required">*</span></label><br/>
-            <input type="text" title="<?php echo Mage::helper('core')->quoteEscape($fieldName) ?>" class="required-entry input-text" id="<?php echo $_code ?>_bolt_reference" name="payment[bolt_reference]" value="<?php echo $this->getInfoData('bolt_reference') ?>"/>
+            <label for="<?php echo $_code ?>_payment_button"><?php echo $fieldName ?> <span class="required">*</span></label><br/>
+            <div class="bolt-checkout-button with-cards <?= $theme; ?>"></div>
+            <input type="hidden" title="<?php echo Mage::helper('core')->quoteEscape($fieldName) ?>" class="required-entry" id="<?php echo $_code ?>_payment_button" name="payment[bolt_reference]" value="<?php echo $this->getInfoData('bolt_reference') ?>"/>
         </div>
     </li>
 </ul>
+
+<script type="text/javascript">
+    (function() {
+
+        //////////////////////////////////////////////////
+        //  JSONP function to reliable load connect.js
+        //  after ajax reloads of this block's template
```

Comment:
Why this is needed?

Reply:
This is the same pattern used on the onepage checkout, which is very similar to the admin page.  So, the quick answer is for reliable convenience. 

But also, it is true that in the past, it was the way to style and attach events to buttons.  It is true that calling BoltConnect could be enough, but to load it with a dynamically assigned publishable key would require rewriting the head of the admin page, creating an additional or overriding block, creating additional templates, a lot of XML configuration, etc. or some similar tricks.  So, need is too strong a word.  

It is done this way because it is easier with little cost, and it is also status quo.